src/plugins/embeddable/public/lib/panel/embeddable_panel.tsx

```diff
@@ -245,11 +245,13 @@ export class EmbeddablePanel extends React.Component<Props, State> {
       new EditPanelAction(this.props.getEmbeddableFactory),
     ];
 
-    const sorted = actions.concat(extraActions).sort((a: Action, b: Action) => {
-      const bOrder = b.order || 0;
-      const aOrder = a.order || 0;
-      return bOrder - aOrder;
-    });
+    const sorted = (actions as Array<Action<{ embeddable: IEmbeddable }>>)
+      .concat(extraActions)
+      .sort((a: Action<{ embeddable: IEmbeddable }>, b: Action<{ embeddable: IEmbeddable }>) => {
```

Comment:
why do we need to provide type for a and b, shouldn't they be deducted from the type of `actions` ?

Reply:
Removed type annotations as it understands them automatically, I guess I aded those annotations before I added the one on line 248, and after adding the one on line 248 these are not necessary anymore.